snuba/cli/subscriptions_scheduler.py

```diff
@@ -72,11 +107,36 @@ def subscriptions_scheduler(
 
     configure_metrics(StreamMetricsAdapter(metrics))
 
+    entity_key = EntityKey(entity_name)
+
+    storage = get_entity(entity_key).get_writable_storage()
+
+    assert (
+        storage is not None
+    ), f"Entity {entity_name} does not have a writable storage by default."
+
+    stream_loader = storage.get_table_writer().get_stream_loader()
+
+    scheduled_topic_spec = stream_loader.get_subscription_scheduled_topic_spec()
+    assert scheduled_topic_spec is not None
+
+    producer = KafkaProducer(
+        build_kafka_producer_configuration(
+            scheduled_topic_spec.topic,
+            override_params={
+                "partitioner": "consistent",
+                "message.max.bytes": 50000000,  # 50MB, default is 1MB
```

Comment:
I don't think a scheduled subscription can get even remotely close to something 3 order of magnitude smaller. I think you can leave it to the default.